kernel/GBEngine/kutil.cc

```diff
@@ -832,7 +835,13 @@ BOOLEAN kTest_T(TObject * T, ring strat_tailRing, int i, char TN)
   {
     if (T->p == NULL && i > 0)
       return dReportError("%c[%d].p is NULL", TN, i);
-    pFalseReturn(p_Test(T->p, currRing));
+    if (currRing->isLPring && T->shift > 0) {
```

Comment:
Bei den Polynomen in T wird nur das Leitmonom geshiftet, deswegen ist das Leitmonom hier von geringster Ordnung.